scripts/move_paper_feed.py

```diff
@@ -49,7 +49,7 @@ def OK_to_add(args, connect, handling):
         print('Error: ', connect.upstream_part, "already connected.")
         is_OK_to_add = False
     # 3 - check to see if antenna is already connected (should be, but isn't necesarily active)
-    is_connected = handling.is_in_connections(connect.downstream_part, 'A', return_active=True)
+    is_connected = handling.is_in_connections(connect.downstream_part, 'P', return_active=True)
```

Comment:
Yes - I have an unpushed branch where I'm updating Docs.  This is one of those to-be-deprecated scripts after this PR.